netty/src/main/java/io/atomix/catalyst/transport/SslProtocol.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License
+ */
+package io.atomix.catalyst.transport;
+
+/**
+ * SSL protocol constants.
+ *
+ * @author <a href="http://github.com/kuujo>Jordan Halterman</a>
+ */
+public enum SslProtocol {
```

Comment:
I created an `SslProtocol` enum to provide constants for specifying SSL protocols to aid in validation of protocol strings. The enum value name - e.g. `SSLv1_1` - is used to indicate the string - `SSLv1.1` - by doing a string replace:
```java
SslProtocol.SSLv1_1.name().replace("_", ".")
```